adminSDK/alertcenter/quickstart/src/main/java/AdminSDKAlertCenterQuickstart.java

```diff
@@ -36,12 +37,13 @@
 
   private static final String APPLICATION_NAME = "Google Admin SDK Alert Center API Java Quickstart";
   private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
   /**
-   * Global instance of the scopes required by this quickstart. If modifying these scopes, delete
-   * your previously saved tokens/ folder.
+   * Global instance of the scopes required by this quickstart.
+   * If modifying these scopes, delete your previously saved tokens/folder.
```

Comment:
There actually is supposed to be a space between these. The folder is named "tokens", and we are instructing that they delete the whole folder.

Reply:
Done.